crates/matrix-sdk/src/sliding_sync/builder.rs

```diff
@@ -75,8 +75,8 @@ impl SlidingSyncBuilder {
     /// Enroll the list in caching, reloads it from the cache if possible, and
     /// adds it to the list of lists.
     ///
-    /// This will raise an error if a [`storage_key()`] was not set, or if there
-    /// was a I/O error reading from the cache.
+    /// This will raise an error if a [`Self::storage_key()`] was not set, or if
```

Comment:
```suggestion
    /// This will raise an error if a [`storage_key()`][Self::storage_key] was not set, or if
```

Reply:
This broke formatting :stuck_out_tongue: